test/functional/p2p_blockfilters.py

```
@@ -280,6 +303,254 @@
+
+        # Add OP_RETURN output
+        lock_tx.vout.append(CTxOut(int(total_locked * COIN), CScript([OP_RETURN, b""])))
+
+        lock_tx.nVersion = 3
+        lock_tx.nType = 8  # asset lock type
+        lock_tx.vExtraPayload = lockTx_payload.serialize()
+
+        # Sign and send the transaction
+        signed_tx = node.signrawtransactionwithwallet(lock_tx.serialize().hex())
+        final_tx = tx_from_hex(signed_tx["hex"])
+        txid = node.sendrawtransaction(final_tx.serialize().hex())
```

Ruff (0.12.2)

Local variable `txid` is assigned to but never used

Remove assignment to unused variable `txid`

(F841)